lms/djangoapps/student_account/views.py

```diff
@@ -427,6 +427,7 @@ def _get_extended_profile_fields():
         "state": _(u"State/Province/Region"),
         "company": _(u"Company"),
         "title": _(u"Title"),
+        "job_title": _("Job Title"),
```

Comment:
I don't know exactly what it affects, but for consistency, I'd make Job Title, Profession, and Specialty unicode strings like the rest of them.

Reply:
Yeah that's fair - I'll make that change

Reply:
@BessieSteinberg please make sure to test that Job Title shows up in the "Additional Information" section of Account Settings. Also make sure to test and update the Site Configuration for Wharton and do your testing with that configuration.